src/main/scala/sha3.scala

```diff
@@ -17,6 +17,10 @@ case object Sha3BufferSram extends Field[Boolean]
  * specification, which differs in the padding behavior
  */
 case object Sha3Keccak extends Field[Boolean]
+/*
+ * Enable specific printf's
```

Comment:
Can you add a comment here saying why. I think it's fine to just say this is for educational/tutorial purposes.